File: examples/UnclaimedDraftEditAndResendExample.cs
using System;
using System.Collections.Generic;
using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

public class Example
{
    public static void Main()
    {
        var config = new Configuration();
        // Configure HTTP basic authorization: api_key
        config.Username = "YOUR_API_KEY";

        // or, configure Bearer (JWT) authorization: oauth2
        // config.AccessToken = "YOUR_BEARER_TOKEN";

        var unclaimedDraftApi = new UnclaimedDraftApi(config);

        var data = new UnclaimedDraftEditAndResendRequest(
            clientId: "1a659d9ad95bccd307ecad78d72192f8",
            testMode: true
        );

        var signatureRequestId = "2f9781e1a83jdja934d808c153c2e1d3df6f8f2f";

        try
        {
            var result = unclaimedDraftApi.UnclaimedDraftEditAndResend(signatureRequestId, data);
            Console.WriteLine(result);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Dropbox Sign API: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: examples/UnclaimedDraftEditAndResendExample.rb
require "json"
require "dropbox-sign"

Dropbox::Sign.configure do |config|
    config.username = "YOUR_API_KEY"
    # config.access_token = "YOUR_ACCESS_TOKEN"
end

unclaimed_draft_edit_and_resend_request = Dropbox::Sign::UnclaimedDraftEditAndResendRequest.new
unclaimed_draft_edit_and_resend_request.client_id = "b6b8e7deaf8f0b95c029dca049356d4a2cf9710a"
unclaimed_draft_edit_and_resend_request.test_mode = false

begin
    response = Dropbox::Sign::UnclaimedDraftApi.new.unclaimed_draft_edit_and_resend(
        "fa5c8a0b0f492d768749333ad6fcc214c111e967", # signature_request_id
        unclaimed_draft_edit_and_resend_request,
    )

    p response
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling UnclaimedDraftApi#unclaimed_draft_edit_and_resend: #{e}"
end
